### DISCUSSION TITLE
support

### DISCUSSION THREAD

**Discussion by lixiaodao1234** [5axes/CustomSupportCylinder#11] [Q&A]
Can the mouse frame select an area to generate a support structure？

**Comment by lixiaodao1234** [comment]
![Snipaste_2021-08-21_10-53-59](https://user-images.githubusercontent.com/82818628/130308390-2cc6d929-cda9-47fc-a9f4-93d07de0cb3e.png)
![Snipaste_2021-08-21_10-54-13](https://user-images.githubusercontent.com/82818628/130308391-db6dcf97-c2d7-4fff-b0c1-fd084508933f.png)

**Reply by 5axes** [reply]
Would be a nice feature, but too complex for me.

**Reply by lixiaodao1234** [reply]
In fact, this is the same as the automatic support structure, automatic support structure through the identification of areas to support, while manual support can be selected by the left mouse button box support area, click the right mouse button to add jagged support structure. I'm reading your source code and I'm going to implement this functionality. I want to write a mouse event function in Python to read the 3d coordinate point box to select the support area
